packages/esm-patient-orders-app/src/order-basket/order-basket.workspace.tsx

```diff
@@ -122,6 +123,16 @@ const OrderBasket: React.FC<DefaultWorkspaceProps> = ({
               inline
             />
           )}
+          {ordersWithErrors.map((order) => (
+            <InlineNotification
+              kind="error"
+              title={t('saveDugOrderFailed', 'Error ordering {{orderName}}', { orderName: order.display })}
+              subtitle={order.extractedOrderError?.fieldErrors?.join(', ')}
+              lowContrast={true}
```

Comment:
```suggestion
              lowContrast
```

Nit: You can safely omit values for props that are meant to be truthy.